drivers/adc/adc_ads1x1x.c

```diff
@@ -20,6 +21,17 @@
 
 LOG_MODULE_REGISTER(ADS1X1X, CONFIG_ADC_LOG_LEVEL);
 
+#if DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1115, alert_rdy_gpios) || \
+	DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1114, alert_rdy_gpios) || \
+	DT_ANY_COMPAT_HAS_PROP_STATUS_OKAY(ti_ads1113, alert_rdy_gpios) || \
```

Comment:
ADS1113 and ADS1013 does not have  ALERT/RDY pins as per the datasheet, please drop those variants if not needed.

Reply:
oops, my bad